src/main/java/com/moodybluez/enterprise/service/IEntryService.java

```diff
@@ -1,16 +0,0 @@
-package com.moodybluez.enterprise.service;
-
-import com.moodybluez.enterprise.dto.Entry;
-
-import java.util.Map;
-
-public interface IEntryService {
-
-    Map<String, Entry> fetchAll();
-
-    void saveEntry(Entry entry);
-
-    Entry fetchByDate(String date);
-
-
-}
```

Comment:
Removed these files, because the logic here should really be in the DAO layer, which I have created and that includes necessary methods created in these service layer files.